recipe/meta.yaml

```diff
@@ -53,7 +52,7 @@ outputs:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
         #- arm-variant * {{ arm_variant_type }}  # [aarch64]
```

Comment:
Now that there are two variants in cuda-cccl:
https://github.com/AnacondaRecipes/cuda-cccl-feedstock/pull/5/files#diff-ff61408cdc05bc9667deeadb55e4aaceb1371972076b6bf6934f9008920f2bd2
Won't we also need arm-variant? https://github.com/conda-forge/arm-variant-feedstock